src/openrct2/rct2/DATLimits.h

```diff
@@ -0,0 +1,21 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2021 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+#include <cstdint>
+
+namespace RCT2::ObjectLimits
```

Comment:
Wouldn't `DATLimits` or `DATObjectLimits` make more sense for the naming?

Reply:
Hmm perhaps. I was imagining that OpenRCT2::ObjectLimits would be the other limits file so thats what the naming was designed around.

Reply:
Ah well, I'm fine with this too.